packages/strapi-generate-api/templates/mongoose/service.template

```diff
@@ -60,12 +56,15 @@ module.exports = {
    */
 
   count: (params) => {
-    // Convert `params` object to filters compatible with Mongo.
-    const filters = strapi.utils.models.convertParams('<%= globalID.toLowerCase() %>', params);
+    const model = <%= globalID %>;
 
-    return <%= globalID %>
-      .countDocuments()
-      .where(filters.where);
+    const filters = convertRestQueryParams(params);
+
+    return buildQuery({
```

Comment:
We should keep the same structure for each example. You create a `const model` for the count method but you don't it for the `fetchAll` method above.